client/state/login/actions.js

```diff
@@ -22,11 +22,22 @@ const loginErrorMessages = {
 	empty_password: translate( 'The password field is empty.' ),
 	empty_two_step_code: translate( 'The verification code field is empty.' ),
 	empty_username: translate( 'The username field is empty.' ),
-	incorrect_password: translate( 'Invalid username or password.' ),
+	incorrect_password: translate( 'Invalid password.' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Invalid password' )` *ES Score: 17*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/login-validation)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).